app/views/layouts/application.html.erb

```diff
@@ -34,6 +34,7 @@
     <%= stylesheet_pack_tag 'application' %>
     <%= javascript_pack_tag 'application' %>
     <%= yield :head %>
+    <script src="//cdn.polyfill.io/v1/polyfill.min.js" async defer></script>
```

Comment:
Why are we pulling this externally versus putting it in our vendored assets?

Reply:
This was mistakenly left in after a test attempt to get `fetch` to work in IE11. It didn't work, so taking it out.